addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -696,6 +696,49 @@ QUnit.test('display command suggestions on typing "/"', async function (assert)
     );
 });
 
+QUnit.test('do not send typing notification on typing "/" command', async function (assert) {
+    assert.expect(2);
+
+    this.data['mail.channel'].records.push({ id: 20 });
+    this.data['mail.channel_command'].records.push({
+        channel_types: ['channel'],
+        help: "List users in the current channel",
+        name: "who",
+    });
+    await this.start({
+        async mockRPC(route, args) {
+            if (args.method === 'notify_typing') {
+                assert.ok(true, "partner should not be notified when typing command");
```

Comment:
Don't make a dummy assert.ok true that relies on assert count, at least use `assert.step` in this case, it's a bit more clear what is expected from them.